src/android/CameraLauncher.java

```diff
@@ -994,6 +994,7 @@ private Bitmap getScaledAndRotatedBitmap(String imageUrl) throws IOException {
                         // read exifData of source
                         exifData = new ExifHelper();
                         exifData.createInFile(filePath);
+                        exif.readExifData();
```

Comment:
Where is this variable used? If `this.correctOrientation` is true, it is overwritten again and then used to get the orientation, but otherwise?